server/assets/javascripts/mfa.js

```diff
@@ -29,6 +29,51 @@ function insertTitle(){
     $("<h1>Authenticator app</h1>").insertBefore("#attributeList");
 }
 
+function hideTryThisLink(){
+    $('a[href^="otpauth://"]').hide();
+}
+
+function showAccountDetails(){
+    hideTryThisLink();
+    const details = getAccountDetails();
+    const accountDetailsHtml = 
+    `<div class="account-details">
+        <p id="detailText">You can enter these account details into the authenticator app manually.</p>
+        
+        <label>Account Name</label>
+        <p id="accountId">${details.externalId}</p>
+
+        <label id="secretLabel">Secret</label>
+        <p id="secret">${details.secret}</p>
+    </div>`
+
+    $(accountDetailsHtml).insertAfter('#attributeList');
+}
+
+function getAccountDetails(){
+    const authDetails = $('#totpQrCodeControl-text').text();
+
+    const details = {};
+    const externalIdMatch = authDetails.match(/hmcts-stg-external-id:([^?]+)/);
```

Comment:
You can't rely on this text as it will be different on each environment

Reply:
Updated regex now relies on 2nd : until 1st ?

Reply:
🤞